nbf/samples/hello_world.nbf

```diff
@@ -0,0 +1,350 @@
+03_0000200001_0000000000000001
+03_0000200002_0000000000000001
+03_0000200001_0000000000000000
+fe_0000000000_0000000000000000
+03_0000208000_0000000000000042
+03_0000208001_00000000000008aa
+03_0000208002_0000000000001842
+03_0000208003_0000000104000099
+03_0000208004_000000007807800e
+03_0000208005_0000000000000196
+03_0000208006_0000000000010040
+03_0000208007_0000000301800001
+03_0000208008_0000000000022852
+03_0000208009_000000000000007a
+03_000020800a_000000000000001e
+03_000020800b_0000000000000405
+03_000020800c_000000010a800954
+03_000020800d_00000000d044048e
+03_000020800e_0000000080000016
+03_000020800f_000000000504040d
+03_0000208010_000000000000004d
+03_0000208011_000000010e8ce0d4
+03_0000208012_000000000154443d
+03_0000208013_00000000000400ae
+03_0000208014_0000000304800001
+03_0000208015_0000000104800844
+03_0000208016_000000010d0000c4
+03_0000208017_000000000044108e
+03_0000208018_0000000000000016
+03_0000208019_0000000304800001
+03_000020801a_000000000044188e
+03_000020801b_0000000000000016
+03_000020801c_0000000304800001
+03_000020801d_000000011284004c
+03_000020801e_000000000000384a
+03_000020801f_000000005044b80e
+03_0000208020_000000000000019e
+03_0000208021_0000000110001041
+03_0000208022_0000000112000444
+03_0000208023_00000000d044888e
+03_0000208024_0000000000000196
+03_0000208025_00000001158000c4
+03_0000208026_000000011480e254
+03_0000208027_000000000002384a
+03_0000208028_0000000316000001
+03_0000208029_000000000001384a
+03_000020802a_0000000316000001
+03_000020802b_000000000006384a
+03_000020802c_00000003170020dc
+03_000020802d_0000000000000046
+03_000020802e_000000031988004c
+03_000020802f_0000000001444445
+03_0000208030_000000004044280e
+03_0000208031_000000000004012e
+03_0000208032_0000000304800001
+03_0000208033_000000000154443d
+03_0000208034_000000032200c04c
+03_0000208035_000000000004012e
+03_0000208036_00000001200000c4
+03_0000208037_000000000001584a
+03_0000208038_00000000484c500e
+03_0000208039_00000000014cdc45
+03_000020803a_000000000000019e
+03_000020803b_000000011d001041
+03_000020803c_000000011f000444
+03_000020803d_00000000c84c088e
+03_000020803e_0000000000000196
+03_000020803f_0000000304800001
+03_0000208040_000000000000584a
+03_0000208041_00000000484c480e
+03_0000208042_00000000014cdc45
+03_0000208043_0000000304800001
+03_0000208044_00000001240000c4
+03_0000208045_0000000125002254
+03_0000208046_00000000000400ae
+03_0000208047_0000000304800001
+03_0000208048_00000000003401ae
+03_0000208049_0000000304800001
+03_000020804a_00000000000c01ae
+03_000020804b_0000000304800001
```

Comment:
I think these lines are all CCE ucode loading, which should be omitted because that component doesn't exist. The nbf script should omit the ucode loading now (I think the RTL is up-to-date enough to have that change).

Reply:
The new script omits the ucode memory flag, which _should_ mean that NBFs it generates omit this; the `hello_world.nbf` here is the old one generated manually a few months ago. I don't want to re-generate and check in a new `hello_world.nbf` because recent perch changes have made memory images an order of magnitude larger (it includes a bunch of unused code). I checked in with Farzam at one point and he said he was aware of it but to my knowledge the linker configuration and/or optimization params haven't been fixed yet. I can manually delete the commands from the file if it'd be clearer.

Reply:
This section has now been removed.

Reply:
https://github.com/black-parrot-sdk/perch/blob/master/Makefile#L8

Should be fixed with latest SDK -- perch comes in a bm variant which doesn't include dramfs initialization, bp-tests default to this